docs/examples/cloudstoragesource/README.md

```diff
@@ -90,6 +90,9 @@ Object Notifications for when a new object is added to Google Cloud Storage
    1. If you are using standard Kubernetes secrets, but want to use a
       non-default one, update `secret` with your own secret.
 
+   1. If you are [managing multiple project](../../install/managing-multiple-projects.md), be careful to specify the `project`
+   where you want your Google Cloud resources resides.
```

Comment:
```suggestion
```suggestion
    1. By default the Storage Notification will be created in the same project as your GKE cluster. However, if you are [managing multiple projects](../../install/managing-multiple-projects.md), then you can specify `spec.project`,
    which is the Google Cloud Project that the Storage Notification is created in.
```
```